tests/test-cases/Group1-Docker-Commands/1-12-Docker-RMI.robot

```diff
@@ -26,15 +26,10 @@ Basic docker pull, restart, and remove image
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${alpine}
     Should Be Equal As Integers  ${rc}  0
 
+    # Gather logs before rebooting
+    Run Keyword And Continue On Failure  Gather Logs From Test Server  -before-reboot-1
```

Comment:
Perhaps we should move this into a keyword `Reboot VCH` that combines the log collection with the `Reboot VM` call - if we always collect logs before rebooting then it will avoid a lot of unnecessary duplication.

Reply:
I'm inclined to defer this change as there are several other tests that use this keyword, and we should make minimal unrelated changes for release purposes. I'll open an issue for this - we can also capture the need to develop a better log bundle naming mechanism that doesn't rely on manual naming for uniqueness across log bundle names.

Reply:
Done: https://github.com/vmware/vic/issues/7143